.github/workflows/stale.yml

```
@@ -13,31 +13,30 @@ name: Mark stale issues and pull requests
 
 on:
   schedule:
-  - cron: "0 0 * * *"
+    - cron: "0 0 * * *"
 
 permissions:
   issues: write
   pull-requests: write  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)